BankexWallet/InterfaceElements/DesignElements/WalletCreationTypeController/WalletCreationTypeController.swift

```diff
@@ -11,12 +11,16 @@ import UIKit
 class WalletCreationTypeController: UIViewController {
 
     @IBOutlet weak var backButton: UIButton!
+    var isBackButtonHidden: Bool?
     
     override func viewWillAppear(_ animated: Bool) {
         super.viewWillAppear(animated)
         if (navigationController?.viewControllers.count ?? 0) > 1 {
             backButton.isHidden = false
         }
+        if let isHidded = isBackButtonHidden {
+            backButton.isHidden = isHidded
```

Comment:
And then you don't need this `if`-statement with typo inside

Reply:
fixed